fragdenstaat_de/fds_donation/templates/admin/fds_donation/donor/donor_total_amount_per_year_filter.html

```diff
@@ -0,0 +1,32 @@
+{% load i18n static %}
+<!-- Custom filter for donor total amount per year -->
+<details data-filter-title="DonorTotalAmountPerYear" open="">
+    <summary>{{ spec.title }}</summary>
+    <ul>
+        <li>
+            <form action="" method="get">
+                <p>
+                    <label for="id_amount">{% trans "Amount from - to:" %}</label>
+                    <input type="text"
+                           id="id_amount"
+                           value="{{ spec.amount_str|escape }}"
```

Comment:
The `escape` filter is confusing. Why would Django's autoescaping not work here?